utils/sample-createspot.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+
+##  This is sample code that will generally show you how to launch a spot instance on aws and leverage the 
+##  automation built into deepracer-for-cloud to automatically start training
+##  Changes required to work:
+##     Input bucket info
+##     Input security group, iam role, and key-name
+
+## BUCKET is the fundamental input. you need to tell it where in s3 your training will be stored
+## just the bucket name.  no S3:// etc
+
+BUCKET=<########>
+
+
+## Fill these out with your custom information if you want to upload and submit to leaderboard.  not required to run
+DR_UPLOAD_S3_BUCKET=########
```

Comment:
This is autodetected if there is only one aws-deepracer...

Reply:
yes ok will remove this line